src/expressions/functions/builtInFunctions.string.ts

```diff
@@ -792,6 +817,14 @@ export default {
 			localName: 'codepoint-equal',
 			namespaceURI: FUNCTIONS_NAMESPACE_URI,
 			returnType: 'xs:boolean?'
+		},
+
+		{
+			argumentTypes: ['xs:string?', 'xs:string'],
```

Comment:
Could we also add the 2nd variant (with flags)?

Reply:
not easily no, I'd like to delay the flags for now and focus on a first push for the 2-arity matches